src/com/company/zadania/obiektowe/zadanie1/Zamowienie.java

```diff
@@ -16,28 +18,64 @@ public Zamowienie(int maksRozmiar) {
     private int ileDodanych = 0;
 
     public void dodajPozycje(Pozycja p) {
+        for (Pozycja value : pozycja) {
+            if (value == null) continue;
+            if (p.getNazwaTowaru().equals(value.getNazwaTowaru())) {
+                value.setIleSztuk(p.getIleSztuk());
```

Comment:
value.addIleSztuk(p.getIleSztuk() jak juz (bardziej obiektowo) albo 
value.setIleSztuk(value.getIleSztuk() + p.ileSztuk) (mneij obiektowo)
albo baaaardzo obiektowo ale przerost formy value.sumIleSztuk(p) { this.ileSztuk += p.getIleSztuk() }